app/move/steps/create.js

```diff
@@ -137,9 +150,14 @@ module.exports = {
       'gender_additional_information',
     ],
   },
-  '/move-date': {
-    pageTitle: 'moves::steps.move_date.heading',
+  '/move-date-range': {
     fields: ['date_from', 'date_to'],
+    ...moveDateStep,
```

Comment:
Probably better to have the spread as the first item, that way you guarantee the properties here will override any defaults set in the shared object.

Reply:
Still think the spread should be the first property in this object, then any overrides for this particular step are defined after.

Currently, if `moveDateStep` contained a `controller` property it would take priority over the one defined in this step.

Reply:
This was already addressed. Not sure why it didn't show in the changes.